docs/agent/flag-ec2-use-windows-prefix-detection.md

```diff
@@ -0,0 +1,21 @@
+# Windows EC2 hostname starting with `EC2AMAZ-`
+
```

Comment:
Per our discussion, remove this page. It will be added to the documentation repo.

Reply:
Done.